src/components/ui/DataGrid/DataGrid.svelte

```diff
@@ -25,47 +26,52 @@
   export function selectAllVisible() {
     gridOptions?.api?.selectAllFiltered();
   }
+  export function redrawRows(params?: RedrawRowsParams<TRowData>) {
```

Comment:
Instead of exporting these functions (selectAllVisible and redrawRows), I wonder if it makes sense to export `gridOptions` and `bind:gridOptions` from the parent? That way parent components have the entire API at their disposal and we don't have to keep adding export functions when we need to call the ag-grid API directly.

Reply:
I'm not sure about this one. It would be nice not having to expose functions all the time, but `gridOptions` and the `api`/`columnApi` values aren't set until after the grid is initialized. Then that puts those check responsibilities on the parent components so `gridOptions?.api?...` will be everywhere instead of just contained within `DataGrid`